docs/developer_guide/getting_started.md

```diff
@@ -14,20 +14,19 @@ Make sure you have Node.js and npm installed; you need them to obtain and use th
 
 ### Create a new Node.js project
 1. Create and enter a new directory to hold the project:
-  ### Linux/Mac
-  1. `mkdir ~/offline_wallet_generator`
-  2. `cd ~/offline_wallet_generator`
-
+  1. `$ mkdir ~/offline_wallet_generator`
+  2. `$ cd ~/offline_wallet_generator`
```

Comment:
They should be `1.1` and `1.2` as you have a `1`before, and a `2.` later.

Reply:
Would `a. `, `b. `, etc. be acceptable?

Reply:
Or you could just make `1. bullet point text` followed by a code block.

1. __Create and enter a new directory to hold the project.__
    ```bash
    mkdir ~/offline_wallet_generator
    cd ~/offline_wallet_generator
    ```
2. __Next bullet point.__